vrk-kpa/suomidigi
# SUOPA-644 My content

To test: 
- `make fresh`
- Log in and go to https://suomidigi.docker.sh/user
- Make sure it looks some what like this:  
![vrk-suomidigi-oct-2019-uusi 2019-11-05 12-50-43](https://user-images.githubusercontent.com/1712902/68201714-fc20de80-ffca-11e9-8436-de2c03777ab8.jpg)
- Log out and go to container: `make shell` and run: `drush uli --uid=7`
- Log in as the user above, go to https://suomidigi.docker.sh/user and see that there are content on "My content"
- Try also going to other user page like https://suomidigi.docker.sh/user/1 and check that the page looks like any normal user page without blocks found in own user page.
